### PR TITLE
Update keep alive of client to 5min to prevent spamming the server

### PR DESCRIPTION
**Pull request by NicolasMahe**
Related to https://github.com/mesg-foundation/core/issues/760 and https://github.com/mesg-foundation/core/issues/759

Client should not ping the server more often than the `EnforcementPolicy.MinTime` set on the server. By default it's 5min, so I set the client to ping the server every 5min.

### CODE REVIEW

File: client/service/service.go [hunk 1]
```diff
@@ -78,7 +78,7 @@ type Option func(*Service)
 // New starts a new Service with options.
 func New(options ...Option) (*Service, error) {
 	dialKeepaliveOpt := grpc.WithKeepaliveParams(keepalive.ClientParameters{
-		Time: 1 * time.Minute,
+		Time: 5 * time.Minute,
```
Comment (krhubert) [inline client/service/service.go]: If `EnforcementPolicy.MinTime` is set to 5 min maybe we should set this a little bit higher, or decrease MinTime on the server?

Comment (NicolasMahe) [reply client/service/service.go]: @krhubert What did we say about client time again?

Comment (antho1404) [reply client/service/service.go]: also using this variable to explain the why 5 could be good

`Time: EnforcementPolicy.MinTime + Time.epsilon`

I don't know if epsilon exists in go but something like that to add a small delay

Comment (NicolasMahe) [reply client/service/service.go]: I think `EnforcementPolicy.MinTime` is enforce for at least 3 ping in this period. So even if `ClientParameters.Time` = `EnforcementPolicy.MinTime` it's find.

Comment (krhubert) [reply client/service/service.go]: EnforcementPolicy.MinTime - 1 min
Time for server - 1 min
Time for client - 2 min

**Comment by NicolasMahe** [comment]
Closing in favor of https://github.com/mesg-foundation/core/pull/774

File: client/service/service.go (hunk 1 above)
Comment (NicolasMahe) [reply client/service/service.go]: See https://forum.mesg.com/t/reduce-keepalive-values/247

**Pull request closed without merging**